rails/commands-pattern/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+
+  def create
+    authorize User
+
+    Users::Create.new(user_params).call.then do |result|
```

Comment:
I think `Create` is not a good example (or at least complete). When using `Update` you have to pass two arguments, and then there are two options:

```ruby
Users::Update.new(user).call(params)
```

or 

```ruby
Users::Update.new(user, params).call
```

I don't know which one to use 🤷‍♀️